packages/components/src/templates/next/components/internal/Navbar/NavItem.tsx

```diff
@@ -64,9 +64,9 @@ export const NavItem = forwardRef<HTMLButtonElement, NavbarItemProps>(
             isExternal={isExternalUrl(url)}
             showExternalIcon={isExternalUrl(url)}
             href={referenceLinkHref}
-            className={item({ isOpen })}
+            className={`${item({ isOpen })} ${groupFocusVisibleHighlight()}`}
```

Comment:
issue: I think the group focus visible highlight might need to be replaced with the non-group version (`focusVisibleHighlight`), as it doesn't work currently (but please verify also):

https://github.com/user-attachments/assets/4a8cfafc-9fd9-4b8d-b58c-0e0c50c171ad

Reply:
@dcshzj address in latest commits, thanks!